spec/amber/controller/base_spec.cr

```diff
@@ -27,6 +27,11 @@ module Amber::Controller
       controller.responds_to?(:client_ip).should eq true
       controller.responds_to?(:request).should eq true
       controller.responds_to?(:response).should eq true
+      controller.responds_to?(:route_action).should eq true
+      controller.responds_to?(:route_resource).should eq true
+      controller.responds_to?(:route_scope).should eq true
+      controller.responds_to?(:route_controller).should eq true
```

Comment:
Do all of these need to be prefixed with `route_`?

Reply:
I prefixed to avoid name collisions with methods define by the user. I am open tomsuggestions

Reply:
I prefixed to avoid name collisions with methods define by the user. I am open tomsuggestions